examples/offscreen.py

```diff
@@ -0,0 +1,35 @@
+"""
+Example demonstrating off-screen rendering.
+"""
+
+import imageio
+import pygfx as gfx
+
+# Instead of a canvas, we provide a size (in logical pixels). Properties
+# like line widths are usually expressed relative to this logical size.
+# We set pixel_ratio to 2 (the default) to double the resolution.
```

Comment:
I guess I don't understand why the default is 2? I would be surprised to get an image twice the size as what I requested

Reply:
Quality ... Note that the `pixel_ratio` property is also used when there is a canvas. On an hdpi screen, its default value is equal to the screen (usually 2), causing the render texture to be twice as large as the logical size. On non-hdpi screens, the default pixel ratio is 2; the render texture is thus of equal size as on an hdpi screen, and ssaa is applied to avoid those jagged edges. We could say that offscreen (without a canvas) the default pixel_ratio is 1, but I think it makes more sense to be consistent, and output high-res images.

Reply:
I don't think anti-aliasing requirements should be on by default, especially SSAA which is the most resource costly approach. But I can get over it if you think this is best :)

Reply:
I understand your concern. The main reasons I think it is a good idea are:
* If you connect an hdpi screen, you're rendering in high resolution. So why not just do that on normal screens too?
* If your making a quick example I'd rather have it look good.

In other words: as it is now, you get about the same quality (and performance penalty) regardless of using a high-res display or not :)

BTW: setting `pixel_ratio` to 0.5 during mouse interaction may be very effective in boosting performance, especially with volume rendering, resulting in 8x less rays to be cast. That last ssaa step will then just smooth the result, so it won't look blocky.

Reply:
You could even time the draw time, and scale `pixel_ratio` to get the highest resolution while still having smooth interaction ;)

Reply:
Fair enough! :)